chef/cookbooks/network/recipes/default.rb

```diff
@@ -393,6 +393,36 @@ def get_datapath_id_for_ovsbridge(bridge)
     nic.tx_offloading = node["network"]["enable_tx_offloading"] || true
   end
 
+  # Do some MTU checks/configuration for the parent of the vlan nic
+  if nic.is_a?(Nic::Vlan)
+    # 1) validate that the parent of vlan nic is not used by a network directly
+    # and the requested mtu for the parent is lower than the vlan nic mtu.
+    # That would not work and setting the mtu on the vlan would fail.
+    networks_using_parent = if_mapping.select { |net, ifaces| ifaces.include? nic.parent }.keys
+    networks_using_parent.each do |net_name|
+      net = Barclamp::Inventory.get_network_by_type(node, net_name)
```

Comment:
@vuntz could it be that I need to get the network via ```Barclamp::Inventory.get_network_definition``` ? Using ```get_network_by_type``` might not return the correct network but in the case of this PR I want to get the correct network definition even if the network is not used on the node.